src/table/columnCheckboxes.component.tsx

```diff
@@ -26,11 +26,10 @@ const ColumnCheckboxes = (props: ColumnCheckboxesProps): React.ReactElement => {
   };
 
   const checkboxes = availableColumns.map((column: Column) => {
-    const header = column.Header?.toString();
     const accessor = column.accessor?.toString();
-    return header && accessor && accessor.toUpperCase() !== 'TIMESTAMP' ? (
+    return accessor && accessor.toUpperCase() !== 'TIMESTAMP' ? (
       <div key={accessor}>
-        <label htmlFor={accessor}>{header}</label>
+        <label htmlFor={accessor}>{accessor}</label>
```

Comment:
So it looks like this change makes the checkbox labels go back to using the accessor, which I think is more likely to be the `systemName`. Would we rather show the user the `userFriendlyName` since this is the title of the column displayed and differs from what the label on the checkbox is?

Reply:
Alternatively, use the `userFriendlyName` as the accessor?

Reply:
I don't think the `userFriendlyName` should be the accessor - as for some this may be undefined and so you have a mix of `userFriendlyName`s and `systemName`s. I'll use the same logic as the headers to display the text for either `userFriendlyName`, `systemName` or `accessor` in that preference order

Reply:
Oh yeah I hadn't thought of that. In that case it's up to you because maybe a mix of `userFriendlyNames` and `systemNames` as labels for checkboxes might be confusing

Reply:
the mixture will only match what the column headers have i.e. the checkboxes will display userFriendlyNames if they exist, and fall back to systemNames (or accessors) if they're undefined.

Also, this component is only here for testing really as this will eventually need overhauling to be the tree structure selector anyway 😅

Reply:
Well true!